### ISSUE TITLE
Scripts cannot run on 4.2.0-beta(491)

### ISSUE THREAD

**Issue by tatsy-ymj** [coteditor/CotEditor#1289]
### Description

Some ruby scripts cannot run on 4.2.0-beta(491).

### To Reproduce

The following script replaces "テスト" with "test".
It works with 4.1.5, but nothing happens with 4.2.0-beta(491).

```
#!/usr/bin/env ruby -Ku
#%%%{CotEditorXInput=AllText}%%%
#%%%{CotEditorXOutput=ReplaceAllText}%%%

while $stdin.gets
  print $_\
            .gsub(/テスト/){"test"}
end
```

### CotEditor version

4.2.0-beta(491)

### macOS version

12.3.1

**Comment by 1024jp** [comment]
Thank you for the report I confirmed the issue.

**Comment by 1024jp** [comment]
Fixed in 71163158519d18dd402d81c4074569dc2440fe00.

**Comment by 1024jp** [comment]
Released CotEditor 4.2.0-beta.2 that fixes this issue.
